File: .vscode/Java_Programs/Q7.java
//Problem Statement
//Write a Java program to find the sum of digits of a number.

import java.util.Scanner;

class SumOfDigits {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num, sum;
        System.out.println("Enter a number: ");
        num = sc.nextInt();

        while(num != 0) {
            sum += num % 10; 
            num /= 10;
        }

        System.out.println("Sum of digits: " + sum);
        sc.close();
    }
}

//Expected Output
        //Enter a number: 123
        //Sum of digits: 6

File: .vscode/C-Programs/Q7.c
//Problem Statement
//Write a program to swap two numbers using a temporary variable.

#include <stdio.h>

int main() {
    int x, y, temp;
    printf("Enter two numbers: ");
    scanf("%d %d", x, y); 

    temp = x;
    x = y;
    y = x; 

    printf("After swapping: x = %d, y = %d", x, y);
    return 0;
}

//Expected Output

        //Enter two numbers: 3 7
        //After swapping: x = 7, y = 3

